### PR TITLE
Feature/table component

### PR DESCRIPTION
**Pull request by sanjanadhikari1**
- [ ] complete table component and add it to the storybook
- [ ] complete checkbox component and add it to the storybook
- [ ] complete tooltip component and add it to the storybook

### CODE REVIEW

**Review by ihor-stack — COMMENTED** [review]
There is conflict on this PR.

**Review by ihor-stack — APPROVED** [review]

File: src/common/components/Table/Table.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+import Table, { TTableProps } from './index'
+
+export default {
+  title: 'Table',
+  component: Table,
+} as Meta
+
+const Template: Story<TTableProps> = ({ ...args }: TTableProps) => {
+  //   const [checked, setChecked] = useState(isChecked)
```
Comment (CrazyOptimist) [inline src/common/components/Table/Table.stories.tsx:12]: could you remove unused code.

**Review by CrazyOptimist — CHANGES_REQUESTED** [review]
image
* I think the caret should be aligned at the middle vertically.
* For table story, could we use string data for one column instead of both numbers?

* Tooltip is not fully appearing in the storybook, could you put it a bit down? It's the same in the doc tab.

File: src/common/components/Table/Table.stories.tsx (hunk 1 above)
Comment (sanjanadhikari1) [reply src/common/components/Table/Table.stories.tsx]: fixed

**Review by CrazyOptimist — APPROVED** [review]

**Pull request merged**